src/test/regress/expected/foreign_key_restriction_enforcement.out

```diff
@@ -1145,15 +1153,14 @@ BEGIN;
 (1 row)
 
 	CREATE TABLE test_table_2(id int PRIMARY KEY, value_1 int, FOREIGN KEY(value_1) REFERENCES test_table_1(id), FOREIGN KEY(id) REFERENCES tt4(id));
-	SELECT create_distributed_table('test_table_2', 'id');
- create_distributed_table
----------------------------------------------------------------------
-
-(1 row)
-
+ERROR:  cannot create foreign key constraint since foreign keys from reference tables and local tables to distributed tables are not supported
```

Comment:
First finish create_distributed and create_reference_tables, then do ALTER TABLE .. ADD FOREIGN KEY

Reply:
When we do 

 ALTER TABLE .. ADD FOREIGN KEY

in the transaction block, it still fails with a different error message this time.

The test here is to "Check that a table with foreign keys to other dist/ref tables cannot be distributed in a transaction block unless you set to run sequentially"

With our changes this test cannot be run because we cannot create a postgres table with foreign keys to dist/ref tables by design. (Which may also be a bug)

So I think removing the coordinator for testing purposes suits here better.